server/handler/base.go

```diff
@@ -174,32 +180,36 @@ func (b *Base) Evaluate(ctx context.Context, installationID int64, trigger commo
 	return b.RequestReviewsForResult(ctx, prctx, client, trigger, result)
 }
 
-func (b *Base) ValidateFetchedConfig(ctx context.Context, prctx pull.Context, client *github.Client, fetchedConfig FetchedConfig, trigger common.Trigger) (common.Evaluator, error) {
+func (b *Base) ValidateFetchedConfig(ctx context.Context, prctx pull.Context, client *github.Client, fc FetchedConfig, trigger common.Trigger) (common.Evaluator, error) {
 	logger := zerolog.Ctx(ctx)
 
-	if fetchedConfig.Missing() {
-		logger.Debug().Msg(fetchedConfig.Description())
+	switch {
+	case fc.LoadError != nil:
+		msg := fmt.Sprintf("Error loading policy from %s", fc.Source)
+		logger.Warn().Err(fc.LoadError).Msg(msg)
 
-		return nil, nil
-	}
+		b.PostStatus(ctx, prctx, client, "error", msg)
```

Comment:
nit (since it seems like `PostStatus` is showing up in this PR): should we have something like `PostErrorStatus` or a constant so that callers don't need to pass around magic `"error"` strings?

Reply:
This is an existing function that is called in slightly different places now. It's also called with `"pending"` and `"success"` (so the possible check statuses in GitHub). We could make defined functions for each status, but I think I'd leave that for a separate PR.